docs/web/docker.md

```diff
@@ -87,43 +89,62 @@ require authentication you have to write a compose file similar to
 [this](../../web/docker/services/docker-compose.psql.yml).
 
 #### PostgreSQL (authentication)
+To run a CodeChecker server and a PostgreSQL database cluster which requires
+authentication you have write a compose file which should be similar to
+[this](../../web/docker/services/docker-compose.psql.auth.yml).
+
 [Docker secrets]((https://docs.docker.com/engine/swarm/secrets/)) can be used
 to define the superuser password in the PostgreSQL instance and to define a
-`.pgpass` file in the CodeChecker server container for database connections.
+`.pgpass` file in the CodeChecker server container for database connections:
 
-Let's create 2 files in your host machine:
+- [`postgres-passwd`](../../web/docker/services/secrets/postgres-passwd): this
+file will contain the superuser password which will be used in the `initdb`
+script during initial container startup. For more information see
+`Docker Secrets` section of the official
+[readme](https://hub.docker.com/_/postgres).
 
-- `/home/$USER/codechecker_secrets/postgres-passwd`: this file will
-contain the superuser password which will be used in the `initdb` script
-during initial container startup. For more information see `Docker Secrets`
-section of the official [readme](https://hub.docker.com/_/postgres).
-E.g.:
-```sh
-mkdir -p /home/$USER/codechecker_secrets/ && \
-echo 'mySecretPassword' > /home/$USER/codechecker_secrets/postgres-passwd && \
-chmod 0600 /home/$USER/codechecker_secrets/postgres-passwd
-```
-- `/home/$USER/codechecker_secrets/pgpass`: this file can contain
+- [`pgpass`](../../web/docker/services/secrets/pgpass): this file can contain
 passwords to be used if the connection requires a password. This file should
 contain lines of the following format:
 `hostname:port:database:username:password`. For more information
 [see](https://www.postgresql.org/docs/9.6/libpq-pgpass.html).
-E.g.:
-```sh
-mkdir -p /home/$USER/codechecker_secrets/ && \
-echo "*:5433:*:$USER:mySecretPassword" > /home/$USER/codechecker_secrets/pgpass && \
-chmod 0600 /home/$USER/codechecker_secrets/pgpass
-```
-
-To run a CodeChecker server and a PostgreSQL database cluster which requires
-authentication you have write a compose file which should be similar to
-[this](../../web/docker/services/docker-compose.psql.auth.yml) and use the
-previously created password files for secrets.
 
 ### Running your app
 Run `docker-compose -f web/docker/services/<service-yml-file> up -d` and
 Compose starts and runs your entire app.
 
+### Deploy your app
```

Comment:
I think mentioning docker swarm here in the title would add some extra information to the reader what to expect in this section.